src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -34,12 +34,12 @@
 /global/cfs/cdirs/kbase/collections/sourcedata/ -> GTDB -> [GTDB_release_version] -> [genome_id] -> genome files
 
 e.g.
-/global/cfs/cdirs/kbase/collections/sourcedata/GTDB -> r207 -> GB_GCA_000016605.1 -> genome files
```

Comment:
I think it would be logical to eliminate the prefix and make the folder name match the NCBI genome_id.

Reply:
So just get rid of the `GB_` and `RS_`? I assume the names are unique without them right? So they're just handy for identifying which genomes are in refseq?

Reply:
Yes, just remove the first 3 chars from the GCDB metadata files. I am uncertain of their specific purpose, but from my observation of the genome IDs, (I checked for r207, r202, and r95), are unique without them. Without them, the genome ids also match what NCBI has.